packages/wagmi/src/utils/provider.ts

```diff
@@ -68,7 +68,7 @@ export function walletConnectProvider<C extends Chain = Chain>({
       return null
     }
 
-    const baseHttpUrl = `${RPC_URL}/v1/?chainId=${NAMESPACE}:${chain.id}&projectId=${projectId}`
+    const baseHttpUrl = `${RPC_URL}/v1/?chainId=${ConstantsUtil.NAMESPACE}:${chain.id}&projectId=${projectId}`
```

Comment:
If this is now coming from "outside of evm package", name for the namespace should be more specific i.e `ConstantsUtil.EIP155`